group-actors/src/main/java/org/sunbird/service/MemberServiceImpl.java

```diff
@@ -215,12 +215,13 @@ public Response handleMemberAddition(
    * @throws BaseException
    */
   @Override
-  public List<MemberResponse> readGroupMembers(String groupId) throws BaseException {
+  public List<MemberResponse> readGroupMembers(String groupId, Map<String, Object> reqContext)
+      throws BaseException {
 
     List<MemberResponse> members = fetchMembersByGroupIds(Lists.newArrayList(groupId));
```

Comment:
add context

Reply:
We use MDC to log traceId in groups, so there is no requirement to pass request context in each method to log traceId